sw/host/ot_certs/src/x509/tests.rs

```diff
@@ -0,0 +1,571 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{bail, Result};
+use base64::{engine::general_purpose, Engine as _};
+use num_bigint_dig::BigUint;
+use std::collections::HashMap;
+use std::iter::zip;
+
+use foreign_types::{ForeignType, ForeignTypeRef};
+use openssl::asn1::{Asn1IntegerRef, Asn1Object, Asn1OctetStringRef, Asn1StringRef, Asn1Time};
+use openssl::ec::EcKey;
+use openssl::pkey::Public;
+use openssl::x509::{X509NameRef, X509};
+
+use crate::offsets::CertificateWithVariables;
+use crate::template::*;
```

Comment:
Please avoid use wildcard includes if possible.

Reply:
I put a wildcard because (a) it is a test, and (b) I am literally using every single type defined in the template, this would be a very long import.